.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "LABEL": {
+      "name": "Title needs formatting",
+      "color": "F9D0C4"
+    },
+    "CHECKS": {
+      "regexp": "^(fix|feat|docs|chore|refactor)(-#[0-9]+)?: .+"
+    },
+    "MESSAGES": {
+      "success": "Thank you! Your PR title meets our guidelines.",
+      "failure": "Thank you for your PR. Unfortunately, it doesn't meet our quality checks. \n\nPlease ensure you follow our contribution guidelines. The PR title should match the required format. \n\nFor example, it should be like: \"fix-#124: Added responsiveness to the Home page screen.\" \n\nOnce you've updated the title to follow the regex format, we can proceed with your PR. Thank you for understanding!"
+    }
```

Comment:
Good one, but in the video why was I not able to find this message when it says title needs formatting?

Reply:
It Will show in the github actions tab @krishnaacharyaa

Reply:
It will show in the github actions tab sir @krishnaacharyaa

Reply:
Can't we make it show in the very same conversation thread? @Ameerjafar

Reply:
Done 👍👍